src/tools/compiletest/src/runtest.rs

```diff
@@ -2042,10 +2050,24 @@ impl<'test> TestCx<'test> {
             if !is_rustdoc {
                 if let Some(ref linker) = self.config.linker {
                     rustc.arg(format!("-Clinker={}", linker));
+                } else {
+                    lld = self
+                        .config
+                        .target_rustcflags
+                        .as_ref()
+                        .map(|f| f.contains("-fuse-ld=lld"))
```

Comment:
```suggestion
                        .map_or(false, |f| f.contains("-fuse-ld=lld"))
```

Reply:
I'll change it but the GH suggestion is not directly applicable since you only highlighted 1 line to change (you can drag to mark more than one line).

Reply:
It may be better to remove this code entirely as suggested in https://github.com/rust-lang/rust/pull/81942#issuecomment-791943800, so this comment may be outdated.